gcloud/monitoring/label.py

```diff
@@ -48,7 +48,7 @@ class LabelDescriptor(object):
     :param description: A human-readable description for the label.
     """
 
-    def __init__(self, key, value_type, description):
+    def __init__(self, key, value_type=LabelValueType.STRING, description=''):
```

Comment:
@supriyagarg and I discussed this off-line and agreed to stick with the move away from named tuples. It adds some verbosity, but it is more familiar to many people, and it seems more flexible.